services/ucs/tests_ucs/integration/test_controller.py

```diff
@@ -35,6 +36,17 @@
 pytestmark = pytest.mark.asyncio()
 
 
+def calc_expected_encrypted_checksum(content: str) -> str:
+    """Calculate the expected checksum for 'encrypted' (test) data in S3.
+
+    This assumes only one object part for simplicity.
+    """
+    part_md5 = hashlib.md5(content.encode(), usedforsecurity=False).hexdigest()
+    concatenated_md5s = bytes.fromhex(part_md5)
```

Comment:
No need to convert it to a hex based representation if that's not used down the line
```suggestion
    part_md5 = hashlib.md5(content.encode(), usedforsecurity=False).digest()
```

Reply:
I admit I was leaning on the copy/paste there, thanks for the tip